### PR TITLE
user.scope and archived meetings

### PR DESCRIPTION
**Pull request by r-peschke**

### CODE REVIEW

File: openslides_backend/action/actions/user/user_scope_permission_check_mixin.py [hunk 1]
```diff
@@ -76,6 +76,7 @@ def get_user_scope(
         Returns the scope of the given user id together with the relevant scope id (either meeting, committee or organization).
         """
         meetings: List[int] = []
+        meetingsd: Dict[int, int]
```
Comment (jsangmeister) [inline openslides_backend/action/actions/user/user_scope_permission_check_mixin.py:79]: please use a better name

File: openslides_backend/action/actions/user/user_scope_permission_check_mixin.py [hunk 2]
```diff
@@ -88,17 +89,27 @@ def get_user_scope(
...
+                GetManyRequest(
+                    Collection("meeting"),
+                    meetings,
+                    ["committee_id", "is_active_in_organization_id"],
+                )
+            ]
+        ).get(Collection("meeting"), {})
+        meetingsd = {
+            meeting_id: odict.get("committee_id")  # type: ignore
+            for meeting_id, odict in result.items()
```
Comment (jsangmeister) [inline openslides_backend/action/actions/user/user_scope_permission_check_mixin.py:103]: same for `odict`

File: openslides_backend/action/actions/user/user_scope_permission_check_mixin.py (hunk 1 above)
Comment (jsangmeister) [reply openslides_backend/action/actions/user/user_scope_permission_check_mixin.py:79]: Also, is the declaration here necessary?

Comment (r-peschke) [reply openslides_backend/action/actions/user/user_scope_permission_check_mixin.py]: > please use a better name

done, not quite sure it is better

Comment (r-peschke) [reply openslides_backend/action/actions/user/user_scope_permission_check_mixin.py]: > Also, is the declaration here necessary?

no, removed

File: openslides_backend/action/actions/user/user_scope_permission_check_mixin.py (hunk 2 above)
Comment (r-peschke) [reply openslides_backend/action/actions/user/user_scope_permission_check_mixin.py]: done

**Pull request merged**